src/components/tile/responsive-cell/responsive-cell.component.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import {
+  FlexBasisProps,
+  FlexGrowProps,
+  JustifyContentProps,
+  MaxWidthProps,
+  PaddingProps,
+} from "styled-system";
+import Box from "../../box";
+import tagComponent, {
+  TagProps,
+} from "../../../__internal__/utils/helpers/tags";
+import StyledDivider from "./responsive-cell.style";
+import Hr from "../../hr/hr.component";
+
+export interface ResponsiveCellProps
+  extends TagProps,
+    PaddingProps,
+    FlexGrowProps,
+    FlexBasisProps,
+    JustifyContentProps,
+    MaxWidthProps {
+  /** The content to render within the responsive cell. */
+  children?: React.ReactNode;
```

Comment:
```suggestion
  children: React.ReactNode;
```

I think we can just make `children` required here